File: service/src/main/java/no/nav/melosys/service/oppgave/dto/PlukkOppgaveInnDto.java
package no.nav.melosys.service.oppgave.dto;

import no.nav.melosys.domain.kodeverk.Sakstemaer;
import no.nav.melosys.domain.kodeverk.Sakstyper;
import no.nav.melosys.domain.kodeverk.behandlinger.Behandlingstema;

public record PlukkOppgaveInnDto(
    Sakstyper sakstype,
    Sakstemaer sakstema,
    Behandlingstema behandlingstema
) {

}

File: service/src/main/kotlin/no/nav/melosys/service/oppgave/dto/PlukkOppgaveInnDto.kt
package no.nav.melosys.service.oppgave.dto

import no.nav.melosys.domain.kodeverk.Sakstemaer
import no.nav.melosys.domain.kodeverk.Sakstyper
import no.nav.melosys.domain.kodeverk.behandlinger.Behandlingstema

@JvmRecord
data class PlukkOppgaveInnDto(
    val sakstype: Sakstyper,
    val sakstema: Sakstemaer,
    val behandlingstema: Behandlingstema
)
